README.md

```diff
@@ -43,6 +45,27 @@ You can test a full setup with Scorpio using Docker compose:
 docker build -t reverseproxy -f Dockerfile-nginx-alpine .
 docker-compose -f scorpio-aaio.yml up
 ```
-Dockerfile-nginx-alpine generates a reverse proxy that will be used on top of the NGSI-LD broker and NGSI-LDES allowing HTTP caching.
+Dockerfile-nginx-alpine generates a reverse proxy that will be used on top of the NGSI-LD broker and NGSI-LDES allowing 
+HTTP caching.
 Note that the environment variables are configurable inside this yaml file.
 Go to `http://localhost:8081` to access the NGSI-LDES DCAT overview through the reverse proxy.
+
+## Project Assumptions
+This contains a list of things to look out for or possibly fix in the following improvements.
+* The Device and DeviceModel LDES’s will NOT contain any data. This is because the NGSI LDES performs a historical 
+request to these specific entities. This is because the data is exposed on the Scorpio Context Broker only once 
+(manually) and there is a retention period of only a few days on historical search.
```

Comment:
This retention period on historical search would be something custom (see comment below)